README.md

```diff
@@ -235,4 +235,25 @@ Then, in your maven settings file, add configuration for the server:
 </servers>
 ```
 
-exactly as you would for any other server configuration.
\ No newline at end of file
+exactly as you would for any other server configuration.
+
+## Authenticating with maven pom.xml
+
+Since version 1.3.XX, you can authenticate using config from the pom itself.
+Just add configuration similar to:
+
+```xml
+ <plugin>
+    <groupId>com.spotify</groupId>
+    <artifactId>dockerfile-maven-plugin</artifactId>
+    <version>1.3.7-SNAPSHOT</version>
+    <configuration>
+        <username>${docker.user}</username>
+        <password>${docker.pass}</password>
```

Comment:
for the example, if the user using a pom like this would have to run `mvn -Ddocker.user=... -Ddocker.pass=...`, but the Mojo now supports properties for `-Ddockerfile.username` and `-Ddockerfile.password`, how about if the example just referred to executing `mvn -Ddockerfile.username=... -Ddockerfile.password=...` so you didn't have to configure `<username>` and `<password>` in the pom at all?

Reply:
Makes sense. We'd have to use the above since we don't control the calling system's command line. I'll add both options